plugin/storage/badger/spanstore/kind.go

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2025 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package spanstore
+
+import (
+	"strconv"
+
+	"github.com/jaegertracing/jaeger/model"
+)
+
+type badgerSpanKind int
+
+const (
+	badgerSpanKindUnspecified badgerSpanKind = iota
```

Comment:
if you are using this as database values, please don't use implicit assignment via iota, give the concrete values that will never change. iota-based enum can change if the order is changed.